docs/docs/configuration/camera_specific.md

```diff
@@ -150,4 +150,4 @@ ffmpeg:
   output_args:
     record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -ar 44100 -c:a aac
     rtmp: -c:v copy -f flv -ar 44100 -c:a aac
-```
\ No newline at end of file
+```
```

Comment:
What's different about this line? Seems like an unrelated change that should be ignored